.github/workflows/ci.yml

```
@@ -18,15 +18,15 @@ jobs:
           git fetch --prune --unshallow
       - name: Install GitVersion
-        uses: gittools/actions/gitversion/setup@v0.6.1
+        uses: gittools/actions/gitversion/setup@v0.9.7
         with:
-          versionSpec: '5.1.x'
+          versionSpec: '5.x'
       - name: Use GitVersion
         id: gitversion
-        uses: gittools/actions/gitversion/execute@v0.6.1
+        uses: gittools/actions/gitversion/execute@v0.9.7
       - uses: actions/setup-java@v1
```

actionlint (1.7.7)

the runner of "actions/setup-java@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)